### DISCUSSION TITLE
NativeAnimationCornerRadius is not applied on samsung device with Android 8.0

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#1115] [General]
NativeAnimationCornerRadius is supposed to be supported from Android M (6.0) and above https://github.com/xamarin/XamarinCommunityToolkit/blob/aba6205274a7fb8c8a55eaedb8001e3a693ed5f1/src/CommunityToolkit/Xamarin.CommunityToolkit/Effects/Touch/PlatformTouchEffect.android.cs#L353 i noticed it is not working on a Samsung device with Android 8.0. I know some Samsung devices used to have issues with ripple effect (i will link a related xf issue later).

Could someone confirm if it is a specific Samsung device issue or it's a TouchEffect/Android issue? By testing on a non samsung device with Android 8.0.

**Comment by AndreiMisiukevich** [comment]
@Cfun1 you probably can run on some emulator... hm
